pkg/arvo/sys/vane/gall.hoon

```diff
@@ -783,9 +837,24 @@
       [p q]:beak
     ::
     ?:  ?=(%raw-poke -.deal)
-      =/  =schematic:ford  [%vale ship-desk +.deal]
-      =/  =note-arvo  [%f %build live=%.n schematic]
-      (mo-pass path note-arvo)
+      ::  TODO: Remove this horrific hack when ford pinto comes!
+      ?+  mark.deal
+        =/  =schematic:ford  [%vale ship-desk +.deal]
+        =/  =note-arvo  [%f %build live=%.n schematic]
+        (mo-pass path note-arvo)
+      ::
+          %chat-action
+        =/  chat-act=(unit chat-action)
+          ((soft chat-action) ;;(chat-action noun.deal))
```

Comment:
```suggestion
          ((soft chat-action) noun.deal)
```

The call to `+soft` replaces the `;;`.  As written this would still crash if the noun didn't match the type.  Feel free to try examples in the dojo.